agent/pyproject.toml

```diff
@@ -38,10 +38,10 @@ minversion = "6.0"
 addopts = "--random-order --cov=lm_agent --cov-report=term-missing --cov-fail-under=70"
 testpaths = ["tests"]
 env = [
-    "LM2_AGENT_AUTH0_DOMAIN = str",
-    "LM2_AGENT_AUTH0_AUDIENCE = str",
-    "LM2_AGENT_AUTH0_CLIENT_ID = str",
-    "LM2_AGENT_AUTH0_CLIENT_SECRET = str",
+    "LM2_AGENT_OIDC_DOMAIN = str",
+    "LM2_AGENT_OIDC_AUDIENCE = str",
+    "LM2_AGENT_OIDC_CLIENT_ID = str",
+    "LM2_AGENT_OIDC_CLIENT_SECRET = str",
```

Comment:
Since these env vars are set by the charm, we'll need to update in the charm as well.

Reply:
sure, I will modify it